.gitlab/container_build/docker_linux.yml

```diff
@@ -109,36 +109,38 @@
     - mkdir -p ${ARTIFACTS_BUILD_CONTEXT}
     - ls $OMNIBUS_PACKAGE_DIR/*.xz | grep -v -- -dbg- | xargs -I '{}' mv '{}' $ARTIFACTS_BUILD_CONTEXT/
 
-.docker_build_job_definition_amd64:
-  extends: .docker_build_job_definition
+.docker_build_amd64:
   image: registry.ddbuild.io/ci/datadog-agent-buildimages/docker_x64$CI_IMAGE_DOCKER_X64_SUFFIX:$CI_IMAGE_DOCKER_X64
   tags: ["arch:amd64"]
   variables:
     ARCH: amd64
 
-.docker_build_job_definition_arm64:
-  extends: .docker_build_job_definition
+.docker_build_arm64:
   image: registry.ddbuild.io/ci/datadog-agent-buildimages/docker_arm64$CI_IMAGE_DOCKER_ARM64_SUFFIX:$CI_IMAGE_DOCKER_ARM64
   tags: ["arch:arm64"]
   variables:
     ARCH: arm64
 
 # build agent7 image
-docker_build_agent7:
-  extends: [.docker_build_job_definition_amd64, .docker_build_artifact]
+.docker_build_agent7:
+  extends: [.docker_build_job_definition, .docker_build_artifact]
   rules:
     - !reference [.except_mergequeue]
     - when: on_success
-  needs:
-    - job: datadog-agent-7-x64
   variables:
     IMAGE: registry.ddbuild.io/ci/datadog-agent/agent
     BUILD_CONTEXT: Dockerfiles/agent
-    TAG_SUFFIX: -7
     TARGET_ARG: --target test
-    BUILD_ARG: --build-arg DD_AGENT_ARTIFACT=datadog-agent-7*-amd64.tar.xz
     CACHE_TARGET: release-base
 
+docker_build_agent7:
+  extends: [.docker_build_amd64, .docker_build_agent7]
+  needs:
+    - job: datadog-agent-7-x64
+  variables:
+    TAG_SUFFIX: -7
```

Comment:
Do we still need to care about agent version?

All jobs seem to include `-7` now, so I wonder if we couldn't have `-7` in the common part, and keep `-fips`/`-jmx` only as the suffix, and rename `docker_build_agent7` to `docker_build_agent`